seed.js

```diff
@@ -104,15 +108,44 @@ const defaultOrderProducts=[
     {orderId:2,productId:2,quantity:1,price:100},
     {orderId:3,productId:3,quantity:1,price:100},
     {orderId:4,productId:4,quantity:1,price:100},
-    {orderId:5,productId:1,quantity:1,price:100}
+    {orderId:5,productId:1,quantity:1,price:100},
+    {orderId:5,productId:5,quantity:1,price:100}
 ];
 
 
 const defaultProducts=[
     {name: 'Super Speed',price: 100,description: 'description',category:['1']},
```

Comment:
Should actually be written to the ProductCategory table according to the associations. So it should be fine as is for now. They are related to each category!